RoslynPluginGenerator/UIResources.resx

```diff
@@ -159,6 +162,14 @@
   <data name="APG_SqaleGeneratedToFile" xml:space="preserve">
     <value>SQALE generated to file {0}</value>
   </data>
+  <data name="APG_TemplateSqaleFileGenerated" xml:space="preserve">
+    <value>
+SQALE: an empty SQALE file for the analyzer was saved to {0}. To provide SQALE remediation information for the analyzer:
+ * rename the file
+ * fill in the appropriate remediation information for each diagnostic
+ * re-run this generator specifying the sqale file to use with the /sqale:[filename] argument.
+</value>
+  </data>
```

Comment:
This is the message that will appear at the end of the run, just before the "jar built" message.